pgxn/neon/neon--1.0.sql

```diff
@@ -32,3 +32,40 @@ CREATE VIEW local_cache AS
 	SELECT P.* FROM local_cache_pages() AS P
 	(pageoffs int8, relfilenode oid, reltablespace oid, reldatabase oid,
 	 relforknumber int2, relblocknumber int8, accesscount int4);
+
+create table postgres_log (
```

Comment:
I've just noticed that users don't have access to these functions and views. 

We're not installing the `neon` extension for users